.automation/build.py

```diff
@@ -2757,7 +2757,7 @@ def manage_output_variables():
                 updated_versions = 1
                 break
         if updated_versions == 1:
-            print("::set-output name=has_updated_versions::1")
+            print('"has_updated_versions=1" >>"$GITHUB_OUTPUT"')
```

Comment:
The Bash changes look correct at a glance, but I suspect printing Bash syntax from Python has no special effect. See EnricoMi/publish-unit-test-result-action#360 for an example of the correct syntax.

Reply:
I'm afraid you're right... I'll rollback the PR to not break behaviour until we have a working solution ( cc  @parkerbxyz  )

Reply:
> I suspect printing Bash syntax from Python has no special effect

In that case, is it working as expected without my changes?

How about something like this? 

```python
import shlex, subprocess

add_to_output_cmd = 'echo "has_updated_versions=1" >>"$GITHUB_OUTPUT"'
add_to_output_args = shlex.split(add_to_output_cmd)
subprocess.run(add_to_output_args)
```

```suggestion
            add_to_output_cmd = 'echo "has_updated_versions=1" >>"$GITHUB_OUTPUT"'
            add_to_output_args = shlex.split(add_to_output_cmd)
            subprocess.run(add_to_output_args)
```